media/css/m24/components/navigation-refresh.scss

```diff
@@ -12,6 +12,24 @@
     }
 }
 
+body:has(.m24-pencil-banner) {
+    .m24-navigation-refresh {
+        .m24-c-navigation-items {
+            &.mzp-is-open {
+                margin-top: 87px;
```

Comment:
**issue:** The approach here works for English, but I think we really have to try and find a way here that does not rely on magic numbers. If the banner copy flows onto three lines for example, the fix no longer works: 

<img width="380" alt="image" src="https://github.com/user-attachments/assets/65a5515a-631c-4fb7-b882-eb71f574b128">

Reply:
**Suggestion:** instead of try to balance showing the banner whilst the mobile navigation menu is open, it might be simpler to just hide it instead:

https://github.com/mozilla/bedrock/blob/main/media/css/m24/components/navigation-refresh.scss#L569-L582

From a user perspective I think this makes sense too, since if someone clicks "menu" then they have shown a clear expression that they are not interested in the banner.

Reply:
I pushed a fix for this to the branch ^